doc/cdi-populators.md

```diff
@@ -0,0 +1,99 @@
+# CDI Volume Populators
+
+## Recommended knowledge
+
+What are [populators](https://kubernetes.io/blog/2022/05/16/volume-populators-beta/)
+
+## Introduction
+CDI volume populators are CDIs implementation of populating PVCs by importing/uploading/cloning data utilizing the new `dataSourceRef` field. New controllers and custom resources for each population method were introduced.
+
+**Values of the new API?**
```

Comment:
Why `?`? I would rename this to something like `Motivation` or `Benefits of using populators`.

Reply:
:) I started with `why using the new API` and changed it. I'll remove the `?` and maybe change values to benefits